### PR TITLE
Streamline content

### PR DESCRIPTION
**Pull request by jeromekelleher**
I've ended up making quite a few changes there in order to try and better address the points that Laurent brought up.

Mainly this has involved bringing stuff from the appendix into the main text - there was a lot of redundancy here anyway.

Probably the controversial thing here is the scope rationale. I think what I've said here is essentially what we've ended up with in practise, but we'd all need to sign up to this one!

There's not as much new content here as it looks from the diff, basically the only new stuff I've written is in the Scope section.

@apragsdale, @molpopgen, @grahamgower, thoughts here?

### CODE REVIEW

**Review by grahamgower — APPROVED** [review]
I like it!

File: paper.tex [hunk 1]
```diff
@@ -286,20 +289,109 @@ \subsection*{Human Data Model}
...
+
+A primary design goal of Demes is to provide a means of
+unambiguously communicating the results of demographic
+model inferences to population genetic simulators.
+Since demography is defined in terms of groups of individuals
+and these groupings are influenced by genetics,
+it is difficult to find a simple definition that separates
+the two. Thus, we have attempted to be pragmatic,
+limiting the features that we include in Demes to
+those that are (or can reasonably be expected to)
```
Comment (apragsdale) [inline paper.tex]: Wording here seems a bit off...

"(or can reasonably be expected to be)"

**Review by apragsdale — APPROVED** [review]
I like this a lot!

**Comment by molpopgen** [comment]
This looks great!  Thanks @jeromekelleher

File: paper.tex (hunk 1 above)
Comment (jeromekelleher) [reply paper.tex]: Yes, a bit jumbled - I've dropped the paranthetical comment.

**Comment by jeromekelleher** [comment]
OK great! Since we all like this I'm going to merge this much and follow up with some more (hopefully small) changes.

**Pull request merged**